ext/search_kit/ang/crmSearchAdmin/crmSearchAdmin.html

```diff
@@ -18,7 +18,7 @@ <h1 crm-page-title>{{ ($ctrl.savedSearch.is_template ? ts('Search Template') : t
       </div>
       <div class="crm-flex-1 form-inline">
         <div class="btn-group">
-          <button type="button" class="btn" ng-class="{'btn-primary': status === 'unsaved', 'btn-warning': status === 'saving', 'btn-success': status === 'saved'}" ng-disabled="status !== 'unsaved'" ng-click="$ctrl.save()">
+          <button type="button" class="btn" ng-class="{'btn-primary': status === 'unsaved', 'btn-warning': status === 'saving', 'btn-success': status === 'saved'}" ng-disabled="status !== 'unsaved'" crm-confirm="$ctrl.confirmSave()" on-yes="$ctrl.save()">
```

Comment:
Small thought: There are common critiques of confirmation dialogs -- they get annoying; people get de-sensitized; etc. That's well and good. 

IMHO, this is not that:

* The call to `$ctrl.confirmSave()` is a computation that decides whether a prompt is worthwhile.
* For most usual edits to regular saved-searches, it will proceed as directed by the user. No need to confirm.
* It only prompts for confirmation if you are deleting or renaming a schema-element (because other things might depend on the schema-element). And it specifically identifies the changed elements.